linkml/generators/jsonldcontextgen.py

```diff
@@ -150,7 +150,8 @@ def visit_slot(self, aliased_slot_name: str, slot: SlotDefinition) -> None:
         else:
             slot_def = {}
             if not slot.usage_slot_name:
-                if slot.range in self.schema.classes:
+                any_of_ranges = [any_of_el["range"] for any_of_el in slot.any_of]
```

Comment:
does this definitely work? All metamodel elements should be accessed as objects, e.g `any_of_el.range`.

Reply:
it worked, but I will change it to `any_of_el.range`